### ISSUE TITLE
Query filter may break because of Interactivity API changes.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-mu-plugins#490]
We use the interactivity API for the [query block](https://github.com/WordPress/wporg-mu-plugins/tree/trunk/mu-plugins/blocks/query-filter). The store parameters are likely to change.

The way we create [the state](https://github.com/WordPress/wporg-mu-plugins/blob/a895fa04096a32bf86243a0920dec34bfc4a0a2a/mu-plugins/blocks/query-filter/src/view.js#L76) will no longer work should the proposal go through.

Instead of creating the store like:

```
store( {
	actions: {
		wporg: {}
} );
```

You will need to pass the namespace

```
store( "myNamespace", {
	actions: {}
)
```

The directive format should also be updated.

**Comment by ryelle** [comment]
Looks like this will happen when we update above GB 17.2 (current is 17.4.1). There are migration instructions in this [changelog discussion](https://github.com/WordPress/gutenberg/discussions/52906#discussioncomment-7810998).

Currently wporg is stuck on 16.8.1 due to a conflict with [Blocks Everywhere](https://github.com/Automattic/blocks-everywhere/issues/192).

**Comment by ryelle** [comment]
Fixed with #561 & #563.
